app/views/RoomsListView/index.tsx

```diff
@@ -458,6 +462,12 @@ class RoomsListView extends React.Component<IRoomsListViewProps, IRoomsListViewS
 			headerTitle: () => <RoomsListHeaderView />,
 			headerRight: () => (
 				<HeaderButton.Container>
+					<HeaderButton.Item
```

Comment:
This looks wrong.
Are we really going to display the troubleshooting on RoomsListView even if there's nothing wrong with push?

Reply:
Yes, in the RoomsListView should appear only needs to alert.